### ISSUE TITLE
Random ad loading 

### ISSUE THREAD

**Issue by Kris4an7** [sbugert/react-native-admob#138]
Hello,
does anyone have an idea what might be causing the ads to show and not show randomly ? 

2 times out for 10 I will have the test ads loaded and the rest it will just remain blank.

Thank you !

**Comment by AriHietamaki** [comment]
Are you getting any errors? I think that I have same issue. I'm getting INTERNAL_ERROR and sometimes NETWORK_ERROR.

**Comment by Kris4an7** [comment]
I fixed it earlier today. 

The problem was that the test ID I was using is probably banned, resulting in the white spaces. 
The random appearance of a Test Banner was probably a bug which added to my confusion. 

I used my own ID together with the a test device set to EMULATOR and it worked fine. I try to be careful so they don't ban my admob account. 

It's strange that I did the exact same thing the previous day and i still got blank spaces, but then I read that it can take several hours for new AdMob Banner ID's to become active. 

I am quite new and I don't want to be out of my place, but I honestly think the documentation can be significantly improved with focus on possible problems. I saw a lot of similar questions in the Issue section most of which with insufficient responses.
